doc/en/components/grids/_shared/excel-style-filtering.md

```diff
@@ -0,0 +1,1017 @@
+           github-src="{ComponentSample}/formatted-filtering-strategy" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-formatted-filtering-strategy"
+           github-src="{ComponentSample}/formatted-filtering-strategy" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
+
+>[!NOTE]
+>The formatted values filtering strategy won't work correctly if you have more than one column bound to the same field from your data and one of the columns has a formatter.
+
+<!-- ComponentStart: TreeGrid -->
+## Tree Filter View
+
+By default, the Excel Style Filtering dialog displays the items in a list view. In order to display them in a tree view you can use the `TreeGridFilteringStrategy` and specify an array of column field names. Filter items will be displayed in a tree view for the speicified columns and in a list view for all other columns. The following sample demonstrates how to show filter items in a tree view for the first column:
+
+<code-view style="height:650px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-tree-filter-view"
+           github-src="{ComponentSample}/tree-filter-view" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+## External Excel Style filtering
+
+As you see at the demos above the default appearance of the Excel Style filtering dialog is inside the {ComponentTitle}. So this dialog is only visible when configuring the filters. There is a way to make that dialog stay always visible - it can be used outside of the grid as a standalone component. In the demo below, the Excel style filtering is declared separately of the {ComponentTitle}.
+
+### Demo
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:620px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-external-excel-style-filtering"
+           github-src="{ComponentSample}/external-excel-style-filtering" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-external-excel-style-filtering"
+           github-src="{ComponentSample}/external-excel-style-filtering" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering"
+           github-src="{ComponentSample}/excel-style-filtering" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.